recipes/surpyvor/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "0.5.0" %}
+
+package:
+  name: surpyvor
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/s/surpyvor/surpyvor-{{ version }}.tar.gz
+  sha256: 58b07eb9848e0b5d38a56f33062bc0e8b06366c58529923cef8aacfc3bd98fd6
+
+build:
+  entry_points:
+    - surpyvor=surpyvor.surpyvor:main
+  noarch: generic
```

Comment:
This should be `noarch: python` instead for pure python packages. `noarch: generic` is for java/perl or packages where you don't compile anything, but only copy the binaries.

Reply:
Alright, does this also apply if there are non-python dependencies?

Reply:
Yes, as long as the main package doesn't use architecture dependent packages to build, then it should be `noarch`, https://docs.conda.io/projects/conda-build/en/latest/resources/define-metadata.html#architecture-independent-packages.